docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -261,6 +261,13 @@ spec:
 ### osFamily (required)
 Operating system on the machine. Permitted values: `bottlerocket`, `ubuntu`, `redhat` (Default: `bottlerocket`).
 
+### osImageURL
+Optional field to replace the default Bottlerocket operating system. EKS Anywhere can only auto-import Bottlerocket. In order to use Ubuntu or Redhat see [building baremetal node images]({{< relref "../../osmgmt/artifacts/#build-bare-metal-node-images" >}})
+to learn more on building and using Ubuntu with an EKS Anywhere cluster. This field is also useful if you want to provide a customized operating system image or simply host the standard image locally.
+
+>**_NOTE:_** If specified for a single Machine Config, osImageURL has to be specified for all the Machine Configs.
```

Comment:
Do you mean "TinkerbellMachineConfig" for Machine Config and "TinkerbellDatacenterConfig" for DatacenterConfig?

Reply:
Yes. I could explicitly call that out if needed. thought it was implied!

Reply:
In general, I like to be specific when possible. Someone could search for TinkerbellMachineConfig and find more information, but it's questionable what they would get searching for Machine Config.